ritasamir/assignment1/Q_01.java

```diff
@@ -0,0 +1,37 @@
+package assignment_1;
+
+import java.util.Arrays;
+
+public class Q_01 {
+
+	public boolean checkAnagram(String firstStr, String secondStr) {
+		String[] array1 = firstStr.split(" ");
+		String[] array2 = secondStr.split(" ");

```

Comment:
optional: Nice algorithm! It will detect single worded anagrams perfectly, as well as sentences made with anagram word pairs. But do you think this method would handle input like: "ab cd" & "a bcd" -> true?

Reply:
I use the space character as separator, Does the two sentences "hello word" & "rowdlleho" anagram or not? is so i don't need to separate the sentences.

Reply:
Depends on the definition you want to use, but I'd say yes. In particular the sentences "hello world" and "hell oworld" are anagrams.
If we want to go even further, you can go down at least two roads:
a) ignore all non letters, so that "hello, world" and "h ello worl d <3" are anagrams
b) don't ignore any characters, so that "hello world" and "hello       world" or "hello,???world!" are NOT anagrams
Again, this really depends on the definition you want to use. The algorithm is fine as it is, it's just more of a fun musing about the subject. :)